lib/plugins/openaps.js

```diff
@@ -226,8 +243,20 @@ function prepOps (prop, sbx) {
   var pump = prop.pump;
   var suggested = prop.openaps && prop.openaps.suggested;
   var suggestedTime = suggested && moment(suggested.timestamp);
-  var enacted = prop.openaps && prop.openaps.enacted;
+
+  var enacted = null;
+  var notEnacted = null;
+
+  if (prop.openaps && prop.openaps.enacted && prop.openaps.enacted.recieved) {
+    enacted = prop.openaps.enacted;
+  }
+
+  if (prop.openaps && !prop.openaps.enacted && prop.openaps.enacted.recieved) {
+    notEnacted = prop.openaps.enacted;
+  }
+
   var enactedTime = enacted && moment(enacted.timestamp);
+  var notEnactedTime = notEnacted && moment(notEnacted.timestamp);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['notEnactedTime' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1658704275/issues/source?bid=2452379&fileBranchId=3033329#l259)